Dokan/Templates/assets.stencil

```diff
@@ -0,0 +1,12 @@
+import UIKit.UIImage
+
+// this is automatic generated file please don't edit it 🔫
+// MARK: - Images
+
+extension UIImage {
+  {% for image in catalogs.first.assets %}
+    static var {{image.name|swiftIdentifier:"pretty"|lowerFirstWord|escapeReservedKeywords}} : UIImage {
```

Comment:
```suggestion

    static var {{image.name|swiftIdentifier:"pretty"|lowerFirstWord|escapeReservedKeywords}} : UIImage {
```

This will allow the result to look like 
```swift
extension UIColor {

    static var accentColor : UIImage {
      UIImage(named: "AccentColor")!
    }

    static var comics22 : UIImage {
      UIImage(named: "comics22")!
    }

    static var image2 : UIImage {
      UIImage(named: "image2")!
    }
}
```